src/cfn/loadCFNStackPolicy.ts

```diff
@@ -0,0 +1,40 @@
+import * as _ from 'lodash';
+import {readFromImportLocation, transform} from '../preprocess';
+import maybeSignS3HttpUrl from './maybeSignS3HttpUrl';
+
+
+// TODO is this used?
```

Comment:
It looks like it is used in `stackArgsToCreateStackInput` for setting the policy during a create (and update?).